doc/src/build/wallet.md

```diff
@@ -27,29 +27,51 @@ in [Genesis customization](#customize-genesis).
 1. Optionally, set `RUST_LOG=debug` for verbose logging.
 1. Initiate `genesis`:
    ```shell
-   sui genesis
+   $ sui genesis
    ```
 
+All of this is contained in configuration and keystore files, as well
+as an `authorities_db` database file.
+TODO: Why do I no longer see a `client_db` file created, only `authorities_db` now?
```

Comment:
I don't have an answer for this but wanted to make sure we do not land this change with a TODO in it (@patrickkuo ?)

Reply:
client_db folder is created by the wallet when it's first started

Reply:
Do you mean when we run this?
wallet --no-shell new-address

Because simply starting wallet doesn't do it.

Reply:
Also, that command assumes the files lives in my home dir:

```
Error: Cannot open wallet config file at "/Users/claymurphy/.sui/sui_config/wallet.conf"
```

So I have to run:

```
claymurphy@Clays-MacBook-Pro tmp % wallet --config wallet.conf --no-shell new-address
```

@awelc how is this easier than having someone manually `mkdir` the directory of their choosing for all of this?

Reply:
I was not aware of this. I would suggest having `client_db`'s default location to be `~/.sui/sui_config/` as well exactly for the the reason @Clay-Mysten mentions - we don't want to force a specific directory creation, but we may want to allow it. We could also make the directory for wallet to be `~/.sui/wallet_config` and have both `wallet.conf`, `wallet.key` and `client_db` to go there by default. @patrickkuo?

Reply:
the wallet will write the `client_db` to the same dir as the `wallet.conf` , it will cause problem if user starts 2 instances of wallet connecting to 2 different network if we hardcode the client_db to `~/.sui/wallet_config`

Reply:
@patrickkuo But in my case, I ran the `wallet new-address` command in my `tmp' directory where `wallet.conf` lives, specifying it with the `--config` flag, only to have a path of:

```
tmp/tmp/client_db
```

That doesn't seem right.

Reply:
See my input and output above.